### PR TITLE
Add coq-fiat-crypto 8.8, 8.9

### PR DESCRIPTION
**Pull request by JasonGross**
Made by copy-paste from 8.7

### CODE REVIEW

**Comment by JasonGross** [comment]
Does the linter no longer support opam 1.2?

```
## Downloading opam 2.0.2 for linux on x86_64...
## Downloaded.
## Where should it be installed ? [/usr/local/bin] ## opam 2.0.2 installed to /usr/local/bin
## Run this script again with '--restore ' to revert.
$ set -o pipefail
$ scripts/opam-coq-list-pr-files | xargs scripts/opam-coq-lint
PR head is: 9dd627d81e460900d7ba75e540dd15c1649431cc
Merge base is: 8051f97c659f318bce46c61c8fbd2fa3a5d20e7e
Ignoring extra-dev/packages/coq-fiat-crypto/coq-fiat-crypto.8.8.dev/descr

Linting extra-dev/packages/coq-fiat-crypto/coq-fiat-crypto.8.8.dev/opam
[WARNING] Running as root is not recommended
/builds/coq/opam-coq-archive/extra-dev/packages/coq-fiat-crypto/coq-fiat-crypto.8.8.dev/opam: Errors.
             error 57: Synopsis and description must not be both empty
```

File: extra-dev/packages/coq-fiat-crypto/coq-fiat-crypto.8.8.dev/opam [hunk 1]
```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+authors: [
+  "Andres Erbsen <andreser@mit.edu>"
+  "Google Inc."
+  "Jade Philipoom <jadep@mit.edu> <jade.philipoom@gmail.com>"
+  "Massachusetts Institute of Technology"
+]
+maintainer: "Matej Košík <matej.kosik@inria.fr>"
```
Comment (gares) [inline extra-dev/packages/coq-fiat-crypto/coq-fiat-crypto.8.8.dev/opam]: wrong, please put your name

File: extra-dev/packages/coq-fiat-crypto/coq-fiat-crypto.8.9.dev/opam [hunk 2]
```diff
@@ -0,0 +1,22 @@
+opam-version: "1.2"
+authors: [
+  "Andres Erbsen <andreser@mit.edu>"
+  "Google Inc."
+  "Jade Philipoom <jadep@mit.edu> <jade.philipoom@gmail.com>"
+  "Massachusetts Institute of Technology"
+]
+maintainer: "Matej Košík <matej.kosik@inria.fr>"
```
Comment (gares) [inline extra-dev/packages/coq-fiat-crypto/coq-fiat-crypto.8.9.dev/opam]: also here

Comment (JasonGross) [reply extra-dev/packages/coq-fiat-crypto/coq-fiat-crypto.8.9.dev/opam]: Updated

File: extra-dev/packages/coq-fiat-crypto/coq-fiat-crypto.8.8.dev/opam (hunk 1 above)
Comment (JasonGross) [reply extra-dev/packages/coq-fiat-crypto/coq-fiat-crypto.8.8.dev/opam]: Updated

**Pull request merged**